### PR TITLE
Devops/docker

### PR DESCRIPTION
**Pull request by emuthiah**
Adding docker-compose and docker file

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -0,0 +1,6 @@
+DIGITALFORMS_BASICAUTH_PASSWORD=myPassword
+DIGITALFORMS_BASICAUTH_USER=myUser
+DIGITALFORMS_SWAGGER_ENABLED=true
+DIGITALFORMS_VIPSORDS_BASEPATH=
+DIGITALFORMS_VIPSORDS_USER=myUser
+DIGITALFORMS_VIPSORDS_PASSWORD=myPassword
```
Comment (shaunmillargov) [inline .env.template:6]: I think you need a new line here.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# environment variables
+**/.env
```
Comment (shaunmillargov) [inline .gitignore:2]: Good, thanks

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      - maven-repository:/root/.m2/repository
+    networks:
+      - rsbc-df-vips
+
+volumes:
+  maven-repository:
+
+networks:
+  rsbc-df-vips:
+    driver: "bridge"
```
Comment (shaunmillargov) [inline docker-compose.yml:29]: New line here too.

File: .env.template (hunk 1 above)
Comment (emuthiah) [reply .env.template]: Added new line. Thanks!

File: docker-compose.yml (hunk 3 above)
Comment (emuthiah) [reply docker-compose.yml]: Added new line. Thanks!

**Pull request merged**